backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/DatabaseHandlerUtil.java

```diff
@@ -703,4 +730,76 @@ public static String trimProjectTag(String projectTag) {
         }
         return projectTag;
     }
+
+    /**
+     * Execute Runnable To Save Attachment To File System
+     */
+    private static void executeRunnableToSaveAttachmentToFileSystem(InputStream content, String userEmail,
+            String documentId, String attachmentId, String fileName) {
+        Runnable fileHandlerRunnable = prepareFileHandlerRunnable(content, userEmail, documentId, attachmentId,
+                fileName);
+        ATTACHMENT_FILE_SYSTEM_STORE_THREAD_POOL.submit(fileHandlerRunnable);
+    }
+
+    /**
+     * Prepare Runnable to save attachment , set permissions and write to file
+     */
+    private static Runnable prepareFileHandlerRunnable(InputStream content, String userEmail, String documentId,
+            String attachmentId, String fileName) {
+        return () -> {
+            try {
+                Path outputDir = Paths.get(ATTACHMENT_STORE_FILE_SYSTEM_LOCATION, userEmail, DOCUMENT_ID + documentId,
+                        ATTACHMENT_ID + attachmentId);
+                Path outputFile = Paths.get(fileName);
+                Path outputFilePath = outputDir.resolve(outputFile);
+                log.info("Preparing to store attachment in file system" + outputFilePath);
```

Comment:
Since store attachment to File system occurs in a separate thread.
I think , if it is `info` then it is a good indication that there has been a process initiated. Instead of not logging anything and silently processing in the background.

However kindly let me know your thoughts , if you strongly recommend it to be `debug` .